src/components/PageElements/Elements/ReviewInlineNewResponse.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react'
+import { Button, Form, Radio, TextArea } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ReviewResponseDecision } from '../../../utils/generated/graphql'
+import { useCreateReviewResponseMutation } from '../../../utils/generated/graphql'
+import { ReviewDetails } from '../../../utils/types'
+
+interface ReviewInlineNewResponseProps {
+  setIsActiveEdit: (_: boolean) => void
+  summaryViewProps: SummaryViewWrapperProps
+  reviewInfo: ReviewDetails
+  stageNumber: number
+}
+
+/*
+This component is visually the same as "ReviewInlineInput", but is used when
+reviewing elements that don't have an applicantResponse. Instead of updating a
+review response, it needs to create a new review response along with its
+associated applicationResponse.
```

Comment:
```suggestion
This component is visually the same as "ReviewInlineInput", but is used when
reviewing elements that don't have an applicantResponse. Instead of updating a
reviewResponse, it needs to create a new one and its associated applicationResponse.
```